src/patternfly/components/DualListSelector/dual-list-selector.scss

```
@@ -257,12 +265,24 @@ $pf-v6-c-dual-list-selector__item--MaxNesting: 10;
 }
 
 .#{$dual-list-selector}__list-item-row {
+  position: relative;
   display: flex;
   font-size: var(--#{$dual-list-selector}__list-item-row--FontSize);
   background-color: var(--#{$dual-list-selector}__list-item-row--BackgroundColor);
 
+  &::before {
+    position: absolute;
+    inset: 0;
+    pointer-events: none;
+    content: "";
+    border-block-start: var(--#{$dual-list-selector}__list-item-row--BorderWidth) solid var(--#{$dual-list-selector}__list-item-row--BorderColor);
+    border-block-end: var(--#{$dual-list-selector}__list-item-row--BorderWidth) solid var(--#{$dual-list-selector}__list-item-row--BorderColor);
```

Just because we might use borders here for something, I'd probably set this up with unique vars? I think it could be like:

```suggestion
    border: solid var(--#{$dual-list-selector}__list-item-row--BorderColor);
    border-block-start-width: var(--#{$dual-list-selector}__list-item-row--BorderBlockStartWidth)
    border-block-end-width: var(--#{$dual-list-selector}__list-item-row--BorderBlockEndWidth);
```